stdlib/src/tkinter.rs

```diff
@@ -0,0 +1,90 @@
+pub(crate) use self::_tkinter::make_module;
+
+#[pymodule]
+mod _tkinter {
+    use crate::builtins::PyTypeRef;
+    use rustpython_vm::function::{Either, FuncArgs};
+    use rustpython_vm::{PyResult, VirtualMachine, function::OptionalArg};
+
+    use crate::common::lock::PyRwLock;
+    use std::sync::Arc;
+    use tk::cmd::*;
+    use tk::*;
+
+    #[pyattr]
+    const TK_VERSION: &str = "8.6";
+    #[pyattr]
+    const TCL_VERSION: &str = "8.6";
+    #[pyattr]
+    const READABLE: i32 = 2;
+    #[pyattr]
+    const WRITABLE: i32 = 4;
+    #[pyattr]
+    const EXCEPTION: i32 = 8;
+
+    fn demo() -> tk::TkResult<()> {
+        let tk = make_tk!()?;
+        let root = tk.root();
+        root.add_label(-text("constructs widgets and layout step by step"))?
+            .pack(())?;
+        let f = root.add_frame(())?.pack(())?;
+        let _btn = f
+            .add_button("btn" - text("quit") - command("destroy ."))?
+            .pack(())?;
+        Ok(main_loop())
+    }
+
+    // TODO: Remove once enough has been implemented.
+    #[pyfunction]
+    fn tk_demo() {
+        let _ = demo();
+    }
```

Comment:
this test function can be removed from this patch and moved to the next patch branch